### PR TITLE
interface: makes the embed button a button

### PR DESCRIPTION
**Pull request by tylershuster**
how to use indigo-react

### CODE REVIEW

**Comment by tacryt-socryp** [comment]
Try using Button instead of Box, despite the difference in visual style.

**Comment by tylershuster** [comment]
I tried. I tried again this morning. The problem is that the button is display: flex and because of how it's written here:
https://github.com/urbit/indigo-react/blob/master/src/buttons/Button.tsx#L187
I can't seem to override that with class `dib`, or `display="inline-flex"` or anything. 

The leap button uses `<Box>` which is why I used it here.

**Comment by matildepark** [comment]
Indigo-react doesn’t use atomic css, it used styled system props, so usually it would just be display=“inline-flex” but I believe the Button component doesn’t take display. I’m unsure if that’s changing next ver. Might be.
—
~haddef-sigwen
https://urbit.org

**Comment by tylershuster** [comment]
Yeah, it doesn't take display. I'm not very good at reading styled-system yet but it *looks* like @g-a-v-i-n has already implemented it here: 

https://github.com/urbit/indigo-react/blob/ga/1.2/src/Button.tsx

**Comment by matildepark** [comment]
The PR itself's visual change, for those of you at home: 
image

File: pkg/interface/src/apps/chat/components/lib/content/url.js [hunk 1]
```diff
@@ -79,11 +80,19 @@ export default class UrlContent extends Component {
           >
             {content.url}
           </a>
-          <a className="bs ml2 f7 pointer lh-copy v-top"
-             onClick={e => this.unfoldEmbed()}
+          <Box
+            border={1}
+            borderRadius={2}
+            display='inline-block'
```
Comment (matildepark) [inline pkg/interface/src/apps/chat/components/lib/content/url.js:86]:
```suggestion
            display='inline-block'
            fontSize={0}
```

image

Comment (tylershuster) [reply pkg/interface/src/apps/chat/components/lib/content/url.js:86]: much better

**Comment by matildepark** [comment]
Also want to note that you can pass `display: 'inline-block'` or the like as a `style` param like cursor; but it has automatic padding. It can break up the line-height rhythm of chat; it is seemingly meant as pure UI, not as part of copy.

image

**Comment by tylershuster** [comment]
ok, the inline thing was kind of a red herring, because I tried the wrong prop. But the padding does foul up the rhythm of chat and the current implementation looks the best for now

**Comment by matildepark** [comment]
Conflict now that the new organisation is in the branch, sorry. -_-;

**Comment by matildepark** [comment]
Might as well take the opportunity to reword the commit above to fit the guidelines. (And this only affects chat, not all of interface).

**Comment by tylershuster** [comment]
great! I'm going to close this and make a new PR when ready

**Pull request closed without merging**